<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_tambahKelas</class>
 <widget class="QDialog" name="form_tambahKelas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSelesai">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>SELESAI</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBatal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>BATAL</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nama</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Jumlah&lt;br/&gt;Siswa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NIP Wali</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNamaKelas">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Identifikator kelas yang dibuat</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nama kelas dapat berupa nomor dan jurusan,
misalnya 11-IPA, 2A, maupun 3 Bahasa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jumlah siswa dapat dihitung dari berapa banyak
siswa yang tergabung dalam suatu kelas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ketahuilah NIP wali kelas yang Anda buat!
Ini berguna saat Anda akan membuat laporan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editJmlSiswa">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Masukan berupa angka</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNIPWali">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Masukan berupa angka</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
